RustConfig

```diff
@@ -0,0 +1 @@
+RUST_CARGO_BUILD_FLAGS="--release"
```

Comment:
[The buildpack docs](https://github.com/emk/heroku-buildpack-rust?tab=readme-ov-file#customizing-build-flags) state that `--release` is the default value of `RUST_CARGO_BUILD_FLAGS`.  Why specify it here?

Reply:
Ah, I must have missed that. I removed that file.